.github/workflows/code-climate-coverage-aggregation.yml

```diff
@@ -119,10 +130,8 @@ jobs:
       
       # Sum the coverage reports
       - name: Summing the coverage reports generated
-        run: ./cc-test-reporter sum-coverage coverage/*-codeclimate.json -p 4 -o coverage/total-codeclimate.json
+        run: ./cc-test-reporter sum-coverage coverage/*-codeclimate.json -p 7 -o coverage/total-codeclimate.json
         
       # Upload total coverage report to Code Climate
       - name: Upload coverage report to Code Climate
-        env:	
-          CC_TEST_REPORTER_ID: ${{secrets.CC_TEST_REPORTER_ID}}  
-        run: ./cc-test-reporter upload-coverage -i coverage/total-codeclimate.json -r $CC_TEST_REPORTER_ID
+        run: ./cc-test-reporter upload-coverage -i coverage/total-codeclimate.json -r env.CC_TEST_REPORTER_ID
```

Comment:
Try removing the -r env.CC_TEST_REPORTER_ID

The documentation seems to indicate it will default it if not provided...

Reply:
trying it